homeassistant/components/mqtt/discovery.py

```diff
@@ -108,9 +108,12 @@ async def async_discovery_message_received(msg: ReceiveMessage) -> None:
         if not (match := TOPIC_MATCHER.match(topic_trimmed)):
             if topic_trimmed.endswith("config"):
                 _LOGGER.warning(
-                    "Received message on illegal discovery topic '%s'. The topic contains "
-                    "not allowed characters. For more information see "
-                    "https://www.home-assistant.io/docs/mqtt/discovery/#discovery-topic",
+                    (
```

Comment:
Funny this seems to make it worse, but maybe the brackets are mandatory now for multi line strings?